apps/package.json

```diff
@@ -117,6 +117,7 @@
     "grunt-webpack": "^1.0.11",
     "hammerjs": "^2.0.8",
     "history": "^2.0.1",
+    "html2canvas": "^0.5.0-beta4",
```

Comment:
It's not ideal that we are using a beta version, but it seems to be more widely used than any of the alternatives.

NodeJS stats:
* 858 downloads in the last day
* 5,311 downloads in the last week
* 21,965 downloads in the last month

This [stackoverflow question](http://stackoverflow.com/questions/10721884/render-html-to-an-image) also lists dom-to-image and server-side PhantomJS as alternatives. dom-to-image won't work for us (see PR description), but server-side PhantomJS could be a viable alternative. 

Another possibility is HTML Renderer, which escaped my initial search: https://theartofdev.com/2014/05/03/generate-image-from-html-using-html-renderer/ I have not looked into this one but it claims to render html into images.

Let me know if you think it is worth looking into any of the other options.